application/config/config.go

```diff
@@ -239,7 +241,9 @@ func (c *Config) loadFile(fileName string) {
 		log.Info().Str("method", "loadFile").Msg("Couldn't load " + fileName)
 		return
 	}
-	defer file.Close()
+	defer func(file *os.File) {
+		_ = file.Close()
+	}(file)
```

Comment:
Why? The error is still discarded

Reply:
Because I get warnings in goland :)

Reply:
If Goland was any good you'd still get them 🤣

Reply:
(jk I love Goland)